crates/celo-revm/src/lib.rs

```diff
@@ -20,7 +20,6 @@ pub mod tx;
 
 pub use api::{
     builder::CeloBuilder,
-    celo_block_env::CeloBlockEnv,
```

Comment:
You can remove `crates/celo-revm/src/api/celo_block_env.rs` after this as well.

Reply:
Ah yes, that must have been lost when splitting up the refactorings into reviewable PRs.